features/dd-sdk-android-session-replay/api/apiSurface

```diff
@@ -1,14 +1,16 @@
 interface com.datadog.android.sessionreplay.ExtensionSupport
   fun getCustomViewMappers(): List<MapperTypeWrapper<*>>
   fun getOptionSelectorDetectors(): List<com.datadog.android.sessionreplay.recorder.OptionSelectorDetector>
-enum com.datadog.android.sessionreplay.ImagePrivacy
+enum com.datadog.android.sessionreplay.ImagePrivacy : PrivacyLevel
   - MASK_NONE
   - MASK_LARGE_ONLY
   - MASK_ALL
 data class com.datadog.android.sessionreplay.MapperTypeWrapper<T: android.view.View>
   constructor(Class<T>, com.datadog.android.sessionreplay.recorder.mapper.WireframeMapper<T>)
   fun supportsView(android.view.View): Boolean
   fun getUnsafeMapper(): com.datadog.android.sessionreplay.recorder.mapper.WireframeMapper<android.view.View>
+interface com.datadog.android.sessionreplay.PrivacyLevel
+fun android.view.View.ddSessionReplayOverrideHidden(Boolean)
```

Comment:
Are we using `dd` instead of `datadog` in the naming on purpose?

Also if we mention `override` in the name, it is not clear what value we overriding. Is it to make what is hidden visible, or the opposite? Maybe just `datadogSessionReplayHide(Boolean)` or something similar?

Reply:
dd was to make the extension a bit shorter, but I can change this. 'Override' is to make it clear that this is one of the privacy overrides, and group with the others that will be in a future pr - image override and text and input override.

Reply:
>'Override' is to make it clear that this is one of the privacy overrides

Why do we need to make it clear? I was thinking that this API is just for the manual control, simply for the user to decide to hide it or not.

Reply:
> dd was to make the extension a bit shorter, but I can change this

Yes, I would rather propose to use `datadog` here. `dd` is not used anywhere in the public API lingvo (apart of the property name for the internal needs of the event models).